src/metabase/server/middleware/security.clj

```diff
@@ -56,7 +56,7 @@
                                      "https://www.google-analytics.com")
                                    ;; for webpack hot reloading
                                    (when config/is-dev?
-                                     "localhost:8080")
+                                     "*:8080")
```

Comment:
This might look scary (accepting any incoming client), but since it's already guarded by `config/is-dev?`, it should be fairly safe.

If this is not deemed safe enough, I can see what I can do to obtain the true local IPv4/v6 address (maybe via `getHostAddress` or something similar).